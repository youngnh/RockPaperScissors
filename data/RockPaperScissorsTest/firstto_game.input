Nate
Ben
R
P
R
P
R
P
