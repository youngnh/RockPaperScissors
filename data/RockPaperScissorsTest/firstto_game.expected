Player 1 Name: 
Player 2 Name: 
[R]ock, [P]aper, or [S]cissors? 
[R]ock, [P]aper, or [S]cissors?
[R]ock, [P]aper, or [S]cissors?
[R]ock, [P]aper, or [S]cissors?
[R]ock, [P]aper, or [S]cissors?
[R]ock, [P]aper, or [S]cissors?
Ben Wins!